xml/System.Numerics.Tensors/ReadOnlyTensorSpan`1+Enumerator.xml

```diff
@@ -42,7 +42,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Enumerates the elements of a <see cref="T:System.Numerics.Tensors.ReadOnlyTensorSpan`1" />.</summary>
```

Comment:
The `TensorSpan` APIs are marked `Experimental` now, should there be an additional annotation in the docs here indicating that so it's surfaced on the page?